pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -679,11 +679,6 @@ def compute_decomposition(wires):
         """
         return [qml.PhaseShift(np.pi / 2, wires=wires)]
 
-    def adjoint(self):
-        op = S(wires=self.wires)
-        op.inverse = not self.inverse
-        return op
-
```

Comment:
I guess maybe I am talking about two separate things. So I agree with removing these definitions, but I was wondering about the list ops in `default.qubit` and why we needed to add `Adjoint(s)` into that list, as opposed to removing it? Since in my mind, its going to ask to expand it in both cases no?